tests/foreman/ui/test_host.py

```diff
@@ -1570,7 +1604,7 @@ def test_global_registration_with_gpg_repo_and_default_package(
 @pytest.mark.tier2
 @pytest.mark.rhel_ver_match('[^6].*')
 def test_global_registration_upgrade_subscription_manager(
-    session, module_activation_key, module_os, rhel_contenthost
+    session, module_activation_key, module_os, module_proxy, rhel_contenthost
```

Comment:
Any reason why adding this parameter? I believe it was deleted as it was not used